tests/asyncunit.nim

```diff
@@ -16,13 +17,6 @@ template asyncSetup*(body: untyped): untyped =
         body
     )())
 
-template asyncTest*(name: string, body: untyped): untyped =
-  test name:
-    waitFor((
-      proc() {.async, gcsafe.} =
-        body
-    )())
-
 template flakyAsyncTest*(name: string, attempts: int, body: untyped): untyped =
```

Comment:
This was removed from libp2p as it didn't work properly. Do you think it's necessary here?

Reply:
removed